### ISSUE TITLE
Font alignment is not upto the mark. 

### ISSUE THREAD

**Issue by Dhanwanth1803** [AnalogAnt/Portfolio#1]
![issue](https://github.com/AnalogAnt/Portfolio/assets/147172285/26908bcf-d9d1-4832-a1df-2b72f190d53e)

**Comment by five-sh** [comment]
Yeah, really didn't expect this from such an esteemed developer who specializes in web dev.  Hoping this will get fixed soon.

**Comment by Dhanwanth1803** [comment]
In addition to this. 
Dont chatgpt the about us section. You need to write it on your own. We as recruiters look at honestly. And you fail. You will be rejected. 


CareersX69.

**Comment by Dhanwanth1803** [comment]
![issue2](https://github.com/AnalogAnt/Portfolio/assets/147172285/0b3ad53a-cdf6-4b3f-8f74-7214a224ed98)
**WHY IS YOUR FACE LIKE THAT?**
